bouncer/src/repo/manipulator/modelconvertor/import/repo_model_import_manager.cpp

```diff
@@ -46,13 +46,12 @@ repo::core::model::RepoScene* ModelImportManager::ImportFromFile(
 		return nullptr;
 	}
 
-	repo::core::model::RepoScene* scene = nullptr;
 	repoTrace << "Importing model...";
-	if (modelConvertor->importModel(file, handler, error)) {
+	repo::core::model::RepoScene* scene = modelConvertor->importModel(file, handler, error);
+	if (scene) {
```

Comment:
This conditional will prevent line 69 from running, no? Also, it would be best not to set the error code to scene fail unless it has actually failed (line 53) - though that code is indeed not used unless scene is null (and we are also moving to handling errors via exceptions), that is an implementation detail so if it ever gets reported somewhere it would be quite confusing!